package.json

```diff
@@ -823,21 +823,21 @@
         }
       },
       {
-        "title": "Checkmarx AI Guided Remediation",
+        "title": "Checkmarx AI Security Champion",
```

Comment:
Can you use the const here too?

Reply:
It's not possible to use the constants.ts file in the package.json file